roles/openstack/neutron/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+# Network defaults
+
+external_network: False
+timeout_network: 180
+shared_network: False
+
+# Subnet defaults
+
+enable_dhcp: True
+dns_servers:
+    - 208.67.222.222
+    - 8.8.8.8
+
+# Router defaults
+
+router_name: router1
+router_network: public
```

Comment:
It's not clear. router is connected to multiple network.
You could also change it to `router_external_network`